File: src/game/level/world1/World1Level.java
package game.level.world1;

import game.entity.enemies.boss.Boss;
import game.entity.models.Enemy;
import game.level.Level;
import game.level.StoryLevel;

public abstract class World1Level extends StoryLevel {
    @Override
    public int getWorldId() {
        return 1;
    }

    @Override
    public Boss getBoss() {
        return null;
    }

    @Override
    public final int getMaxDestroyableBlocks() {
        return 10;
    }


}

File: src/game/level/world1/World1Level.kt
package game.level.world1

import game.entity.enemies.boss.Boss
import game.level.StoryLevel

abstract class World1Level : StoryLevel() {
    override val worldId: Int
        get() {
            return 1
        }

    override val boss: Boss?
        get() {
            return null
        }

    override val maxDestroyableBlocks: Int
        get() {
            return 10
        }
}